commands/eval.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
 		hidden: false,
 		permissions: ["VIEW_CHANNEL", "SEND_MESSAGES"]
 	},
-	do: async (message, client, args, Discord) => {
```

Comment:
`Discord` is included in order for easily evaling with `Discord.whatever()` without having to first require it